src/components/contingency-list.js

```diff
@@ -179,37 +173,40 @@ const ContingencyLists = () => {
     const selectedTheme = useSelector((state) => state.theme);
 
     const contingencyLists = useSelector((state) => state.contingencyLists);
-    const [currentItemType, setCurrentItemType] = useState(null);
-    const [currentItemName, setCurrentItemName] = useState(null);
+    const [currentItem, setCurrentItem] = useState(null);
     const [currentScriptContingency, setCurrentScriptContingency] = useState(
         null
     );
     const [currentFiltersContingency, setCurrentFiltersContingency] = useState(
         null
     );
+
+    //TODO to be removed use list name instead
     const [selectedIndex, setSelectedIndex] = useState(null);
 
     const [btnSaveListDisabled, setBtnSaveListDisabled] = useState(true);
 
     const [aceEditorContent, setAceEditorContent] = useState('');
 
-    const [newListCreated, setNewListCreated] = useState(false);
-    const [newListName, setNewListName] = useState(null);
-
     const [alertEmptyList, setAlertEmptyList] = useState(true);
     const [anchorEl, setAnchorEl] = React.useState(null);
 
     const [openPopupNewList, setOpenPopupNewList] = useState(false);
     const [openPopupRenameList, setOpenPopupRenameList] = useState(false);
-    const [openPopupInfo, setOpenPopupInfo] = useState(false);
     const [openPopupConfirmDelete, setOpenPopupConfirmDelete] = useState(false);
 
-    const [equipmentID, setEquipmentID] = useState('*');
-    const [equipmentName, setEquipmentName] = useState('*');
-    const [equipmentType, setEquipmentType] = useState(equipmentTypes.LINE);
-    const [nominalVoltageOperator, setNominalVoltageOperator] = useState('=');
-    const [nominalVoltage, setNominalVoltage] = useState('');
-    const [countries, setCountries] = useState([]);
+    const emptyFiltersContingency = {
```

Comment:
move this outside of the render() function to avoid rerenders

Reply:
it's already outside of render function